codegen/facelift/templates/ServiceIPC.template.h

```diff
@@ -280,6 +280,14 @@ class {{interface}}IPCProxy : public facelift::IPCProxy<{{interface}}PropertyAda
         int {{property.name}}Size;
         msg >> {{property.name}}Size;
         m_{{property.name}}.setSize({{property.name}}Size);
+
+        {% elif property.type.is_model %}
+        qFatal("Model not supported");
+        {% elif property.type.is_interface %}
+        QString {{property.name}}_objectPath;
```

Comment:
I wouldn't mix this patch with additional features (interface support)

Reply:
fixed